src/store/index.js

```diff
@@ -147,6 +147,13 @@ export default new Vuex.Store({
     toggleNight: toggle("isNight"),
     toggleGrimoire: toggle("isPublic"),
     toggleImageOptIn: toggle("isImageOptIn"),
+    toggleRecordVoteHistory(state, param) {
```

Comment:
you could use the generic toggle function here, toggle("recordVoteHistory")

Reply:
The generic toggle https://github.com/bra1n/townsquare/blob/main/src/store/index.js#L47 is for grimoire, but this is a property of session. (Let me know if that wasn't what you meant here! Since this was the only case wanting session, I decided it was clearer to be explicit than add more abstraction into the generic one.)

Reply:
good point,  I think this mutation should be in session.js.  That file still does not have a toggle function, but it is where I would add it.  Other boolean session properties currently are just using the set function.

Reply:
i've moved it